css/modules/styles.css

```diff
@@ -3592,6 +3592,13 @@ div.ui-memblist-usr .userVideos {
   left: 28px;
 }
 
+#virtualclassCont.congrea #virtualclassAppFooterPanel #fullScreenButton .fullScreen,
+#virtualclassCont.congrea #virtualclassAppFooterPanel #fullScreenExitButton span {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector "#virtualclassCont.congrea #virtualclassAppFooterPanel #fullScreenExitButton span" to come before selector "#virtualclassCont.congrea.playMode #popupContainer.playPopup #loadingWindowCont .askToPlay span" (no-descending-specificity)](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)